Author: Michael Tokarev <mjt@tls.msk.ru>
Description: Simplify mdmonitor.service (Closes: #764647).
 There isn't much for customization for mdadm --monitor.
 it'll just do what it's supposed to do, so just run it.

diff -Naurp mdadm.orig/systemd/mdmonitor.service mdadm/systemd/mdmonitor.service
--- mdadm.orig/systemd/mdmonitor.service
+++ mdadm/systemd/mdmonitor.service
@@ -11,7 +11,4 @@ DefaultDependencies=no
 Documentation=man:mdadm(8)
 
 [Service]
-Environment=  MDADM_MONITOR_ARGS=--scan
-EnvironmentFile=-/run/sysconfig/mdadm
-ExecStartPre=-/usr/lib/mdadm/mdadm_env.sh
-ExecStart=BINDIR/mdadm --monitor $MDADM_MONITOR_ARGS
+ExecStart=BINDIR/mdadm --monitor --scan
